src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs

```diff
@@ -46,6 +46,8 @@ public static partial class PlatformDetection
             GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 16215;
         public static bool IsWindows10Version16251OrGreater => 
             GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 16251;
+        public static bool IsWindows10Version16256OrGreater => IsWindows &&
+           GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 16256;
```

Comment:
I think you'll have to add this to the Unix and Non-OS files because it is added into the ref so the build will fail. Also when building the tests against other OS it will not find this property.

Reply:
Good point, I'll fix that